### PR TITLE
feat(navbar): user account button

### PR DESCRIPTION
**Pull request by mentesniker**
## Changes

The navbar was updated in order to have an usser account button.

## Implementation details

Changes were introduced to the .scss file and to the index.ts file. In the index.ts file, another element was added to the unordered list of the navbar. In the index.scss file several classes were added.

## Screenshots
Before:
image

After:
image

### CODE REVIEW

File: components/navbar/index.ts [hunk 1]
```diff
@@ -173,6 +174,11 @@ export class QiskitNavbar extends LitElement {
                 </li>
               `
             )}
+            <li href="/account" class="navbar__menu__account">
```
Comment (abdonrd) [inline components/navbar/index.ts:177]: An `li` doesn't have a href attribute.

File: components/navbar/index.ts [hunk 2]
```diff
@@ -173,6 +174,11 @@ export class QiskitNavbar extends LitElement {
...
+              <a href="/account" class="navbar__menu__account__link">
```
Comment (abdonrd) [inline components/navbar/index.ts:178]: Extract the account link as the `_homeLink`.

**Review by abdonrd — CHANGES_REQUESTED** [review]
Thanks!

File: components/navbar/index.ts (hunk 1 above)
Comment (mentesniker) [reply components/navbar/index.ts]: Sorry, my bad...

File: components/navbar/index.ts (hunk 2 above)
Comment (mentesniker) [reply components/navbar/index.ts]: I created a new object called _useLink

File: components/navbar/index.scss [hunk 3]
```diff
@@ -40,6 +40,33 @@ $nav-item--spacing-x: $spacing-06;
   border-bottom: $nav-item--border;
   font-family: carbon--font-family("sans");
 
+  &__menu__account {
+    display: inline-flex;
+    max-width: 20rem;
+    background-color: #6929c4;
```
[the remark this answers is not in the document]
Comment (mentesniker) [reply components/navbar/index.scss]: Done!

File: package.json [hunk 4]
```diff
@@ -41,24 +41,23 @@
     "@carbon/layout": "^10.37.1",
     "@carbon/type": "^10.44.0",
     "carbon-web-components": "^1.21.0",
-    "lit": "^2.2.4",
```
Comment (abdonrd) [inline package.json:44]: Please, don't change the `package.json` & `package-lock.json`.

File: components/navbar/index.ts [hunk 5]
```diff
@@ -21,6 +21,7 @@ import { qiskitLogoIcon } from '../icons/qiskit-logo.js';
 import { slackIcon } from '../icons/slack.js';
 import { twitterIcon } from '../icons/twitter.js';
 import { youtubeIcon } from '../icons/youtube.js';
+import { userIcon } from './icons/user.js';
```
Comment (abdonrd) [inline components/navbar/index.ts:24]: Now you can move it to the `components/icons/` folder with the other icons.

**Review by abdonrd — CHANGES_REQUESTED** [review]
Thanks again @mentesniker! 👏

File: package.json (hunk 4 above)
Comment (mentesniker) [reply package.json]: Done!
Sorry, they were added when I brought the changes from main.

**Review by abdonrd — APPROVED** [review]

**Review by korgan00 — APPROVED** [review]
LGTM

**Pull request merged**